.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: minimal
+
+services:
+  - docker
+
+env:
+  - HAZELCAST_IMAGE=hazelcast-enterprise
+  - HAZELCAST_IMAGE=hazelcast-oss
```

Comment:
Doesn't this overwrite the entry above?

Reply:
Nope, this starts 2 jobs - each with one of the `HAZELCAST_IMAGE` values.

Reply:
That's very unintuitive and very cool at the same time :)